features/setting_constraints/matching_arguments.feature

```diff
@@ -47,10 +47,10 @@ Feature: Matching arguments
       |     expected: (1, anything, /bar/)                              |
       |          got: (1, nil, "other")                                 |
 
-  Scenario: Using a RSpec matcher
+  Scenario: Using any matcher from rspec-expectations
     Given a file named "rspec_matcher_spec.rb" with:
       """ruby
-      RSpec.describe "Using a RSpec matcher" do
+      RSpec.describe "Using any matcher from rspec-expectations" do
```

Comment:
The original is correct, mocks matchers are also available here, as is any matcher from any library